plugins/scaffolder/src/next/TemplateWizardPage/Stepper/Stepper.tsx

```diff
@@ -93,7 +97,7 @@ export const Stepper = (props: StepperProps) => {
     const returnedValidation = await validation(formData);
 
     const hasErrors = Object.values(returnedValidation).some(i => {
-      return i.__errors.length > 0;
+      return i.__errors?.length! > 0 ?? false;
```

Comment:
```suggestion
     i.__errors?.length
```

Missing the explicit `> 0` check is a bit meh, but feeling it's worth it here